index.html

```diff
@@ -5,7 +5,7 @@
   <meta name="viewport" content="width=device-width, user-scalable=no">
   <title>OpenRCT2</title>
   <link rel="stylesheet" href="assets/css/index.css">
-  <script src="assets/js/index.js" type="text/javascript" async></script>
+  <script src="assets/js/index.js" type="text/javascript"></script>
```

Comment:
I disagree with this. It makes the JS file a blocking resource. Please drop this commit.